frontend/app/views/offer/offersandcomps.scala.html

```diff
@@ -0,0 +1,50 @@
+@(membersOnlyContent: Seq[model.MembersOnlyContent], noResultText: String)
+
+@main("Offers & Competitions") {
+    <main role="main" class="l-constrained">
+        <section class="listing-header">
+            <h1 class="listing-headline">Offers and competitions</h1>
+        </section>
+        <div class="event-blocks">
```

Comment:
Bit wary of piggybacking on the `event-` classes here. Might need to revisit this soon, either rename these classes to be more generic, or split these components out a bit more. Feels like a lot of this could just be renamed to make it more generic.